mongo/datadog_checks/mongo/mongo.py

```diff
@@ -1009,15 +1015,13 @@ def total_seconds(td):
 
             if ol_options:
                 try:
-                    oplog_data['logSizeMB'] = round(
-                        ol_options['size'] / 2.0 ** 20, 2
-                    )
+                    logSizeMB = ol_options['size'] / 2.0 ** 20
+                    oplog_data['logSizeMB'] = float(Decimal(logSizeMB).quantize(Decimal("2.0")))
```

Comment:
should this be `Decimal("0.01")` to round to the second decimal ?

Reply:
I'm hopefully going to update this soon to use a utility method.